applications/GeoMechanicsApplication/tests/cpp_tests/test_coupling_matrix.cpp

```diff
@@ -0,0 +1,112 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Gennady Markelov
+//
+
+#include "containers/model.h"
+#include "custom_utilities/transport_equation_utilities.hpp"
+#include "includes/checks.h"
+#include "testing/testing.h"
+#include <boost/numeric/ublas/assignment.hpp>
+
+using namespace Kratos;
+
+namespace Kratos::Testing
+{
+
+KRATOS_TEST_CASE_IN_SUITE(CalculateCouplingMatrix2D3NGivesCorrectResults, KratosGeoMechanicsFastSuite)
+{
+    constexpr SizeType voigt_size = VOIGT_SIZE_2D_PLANE_STRESS;
```

Comment:
For our geomechanics continua, we use plane strain, axisymmetric and 3D.
Plane stress is o.k. for this testing but unconventional for this geomechanics.

size_D refers to the spatial dimension here, size_N to the number of nodes, their product is the number of displacement D.O.F.

Reply:
Thank you very much for the clarification.